docs/_docs/reference/metaprogramming/inline.md

```diff
@@ -105,7 +105,8 @@ would typecheck.
 
 Inline methods can be recursive. For instance, when called with a constant
 exponent `n`, the following method for `power` will be implemented by
-straight inline code without any loop or recursion.
+straight inline code without any loop or recursion. It is worth to note that the number of successive
+inlines is limited and can be modified by the compiler setting `-Xmax-inlines`.
```

Comment:
Maybe we should add how nested it can be to make it obvious and not introduce uncertainty.

```suggestion
inlines is limited to 32 and can be modified by the compiler setting `-Xmax-inlines`.
```